src/App.js

```diff
@@ -1,24 +1,37 @@
 import React from 'react';
-import logo from './logo.svg';
 import './App.css';
+import Nav from './Components/Navbar/navBar';
+import Posts from './Components/Posts/Posts'
+import Contacts from './Components/Contacts/Contacts';
+import Users from './Components/UserProfile/userProfile.js'
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <div className="Navbar">
```

Comment:
The usual format for a navbar component would be just one component that holds all the links to the other parts of the website